### PR TITLE
Task #1462890: write tests for GetTags

### PR DESCRIPTION
**Pull request by shenhavba**

### CODE REVIEW

File: sdk/file_sdk/XMPFileFormat/xmp_file_format.cpp [hunk 1]
```diff
@@ -8,7 +8,15 @@ XMPFileFormat::XMPFileFormat(shared_ptr<IStream> file, const string& extension)
...
+const vector<Tag> XMPFileFormat::ReadTags() {
+  return mTags;
+}
+
+string XMPFileFormat::GetOriginalExtension() {
+  throw std::exception("not implemnted");
+}
+
+void XMPFileFormat::Commit(shared_ptr<IStream> file, string& newExtension) {
   throw std::exception("not implemnted");
```
Comment (vakuras) [inline sdk/file_sdk/XMPFileFormat/xmp_file_format.cpp]: typo :) #Resolved

File: sdk/file_sdk/DefaultFormat/default_format.cpp [hunk 2]
```diff
@@ -8,7 +8,15 @@ DefaultFormat::DefaultFormat(shared_ptr<IStream> file, const std::string& extens
 {
 }
 
-const vector<Tag>& DefaultFormat::ReadTags() {
+const vector<Tag> DefaultFormat::ReadTags() {
+  throw std::exception("not implemnted");
+}
+
+string DefaultFormat::GetOriginalExtension() {
+  throw std::exception("not implemnted");
```
Comment (vakuras) [inline sdk/file_sdk/DefaultFormat/default_format.cpp]: you can implement this by default to return the extension.
only in pfile it returns something else #Resolved

**Review by vakuras — APPROVED** [review]
-

File: sdk/file_sdk/XMPFileFormat/xmp_file_format.cpp (hunk 1 above)
Comment (shenhavba) [reply sdk/file_sdk/XMPFileFormat/xmp_file_format.cpp]: Not my typo, i copied it...
will fix in all places

---
In reply to: [135754201](https://github.com/AzureAD/rms-sdk-for-cpp/pull/136#discussion_r135754201) [](ancestors = 135754201)

**Pull request merged**